ui/src/app/utils/navigation.tsx

```diff
@@ -17,6 +17,7 @@ export const currentConceptSetStore = new BehaviorSubject<ConceptSet>(undefined)
 export const urlParamsStore = new BehaviorSubject<any>({});
 export const queryParamsStore = new BehaviorSubject<any>({});
 export const routeConfigDataStore = new BehaviorSubject<any>({});
+export const configDataStore = new BehaviorSubject<any>({});
```

Comment:
nit: naming convention would be `configStore` or `serverConfigStore`. The above variable is a "routeConfigData" "store", not a "routeConfig" "dataStore".